configs/jsactions/rollup.config.js

```diff
@@ -58,30 +60,31 @@ export default async args => {
                     ? command([
                           async () => {
                               if (!isWeb) {
-                                  await Promise.all(
-                                      clientDependencies.map(async dependency => {
-                                          await copyJsModule(
-                                              dirname(require.resolve(`${dependency}/package.json`)),
-                                              join(join(outDir, "node_modules"), dependency)
-                                          );
-                                      })
-                                  );
+                                  if (args.configProject === "nativemobileresources") {
+                                      // `fbjs/lib/invariant` is being used silently by @react-native-community/cameraroll; it is not listed as a dependency nor peerDependency.
+                                      // https://github.dev/react-native-cameraroll/react-native-cameraroll/blob/7c269a837d095a2cb5f4ce13b54ab3060455b17f/js/CameraRoll.js#L14
+                                      const path = join(outDir, "node_modules", "fbjs", "lib");
+                                      mkdirSync(path, { recursive: true });
+                                      await copyAsync(
+                                          join(dirname(require.resolve("fbjs")), "lib", "invariant.js"),
+                                          join(path, "invariant.js")
+                                      );
+                                  } else if (args.configProject === "nanoflowcommons") {
+                                      // `invariant` is being used silently by @react-native-community/geolocation; it is not listed as a dependency nor peerDependency.
+                                      // https://github.dev/react-native-geolocation/react-native-geolocation/blob/1786929f2be581da91082ff857c2393da5e597b3/js/implementation.native.js#L13
+                                      await copyAsync(
+                                          dirname(require.resolve("invariant")),
+                                          join(outDir, "node_modules", "invariant")
+                                      );
+                                  }
```

Comment:
Just to check whether I understand correctly: 
1. before we were copying the entirety of `fbjs` and `invariant` were always copied. Now, we only include `fbjs/lib/invariant.js` with nativemobileresources and `fbjs/lib/invariant` with nanoflowcommons?
2. Does this mean that `invariant` is also removed entirely from nativemobileresources? (wasn't in your "Relevant Changes" section)

Reply:
Before, for both Native Mobile Resources *and* Nanoflow Commons, both `fbjs` and `invariant` (modules) were copied. 

Now, based on their respective needs, Native Mobile Resources has `fbjs/lib/invariant.js` (one file) copied, and Nanoflow Commons has the `invariant` module copied.